clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -264,6 +264,13 @@ scheduler_tmpl_t<memref_t, reader_t>::record_type_is_marker(memref_t record,
     return true;
 }
 
+template <>
+bool
+scheduler_tmpl_t<memref_t, reader_t>::record_type_is_non_marker_header(memref_t record)
+{
+    return false;
```

Comment:
Remind the reader that non-marker headers are always hidden in the memref_t stream.

Reply:
Done.